upgrade_policy/quorum_upgrade/tests/quorum_upgrade_policy_test.move

```diff
@@ -354,6 +354,359 @@ module quorum_upgrade_policy::quorum_upgrade_policy_test {
         test::end(test);
     }
 
+    #[test]
+    fun quorum_upgrade_propose_upgrade_v2_ok() {
+        let test = test::begin(@0x1);
+        let digest: vector<u8> = x"0123456789";
+        let quorum_upgrade_cap = get_quorum_upgrade_cap(3, 5, &mut test);
+
+        test::next_tx(&mut test, @0x1);
+        let proposed_upgrade = quorum_upgrade_policy::propose_upgrade_v2(&quorum_upgrade_cap, digest, ctx(&mut test));
+        quorum_upgrade_policy::share_upgrade_object(proposed_upgrade);
+
+        test::next_tx(&mut test, @0x1);
+        quorum_upgrade_policy::make_immutable(quorum_upgrade_cap);
+        test::end(test);
+    }
+
+    #[test]
+    #[expected_failure(abort_code = quorum_upgrade_policy::quorum_upgrade_policy::EInvalidProposalForUpgrade)]
+    fun quorum_upgrade_authorize_upgrade_v2_bad_cap() {
+        let test = test::begin(@0x1);
+        let digest: vector<u8> = x"0123456789";
+        let quorum_upgrade_cap = get_quorum_upgrade_cap(3, 5, &mut test);
+
+        test::next_tx(&mut test, @0x1);
```

Comment:
It's generally recommended to create constants to hold the addresses
e.g. `const ADDRESS_1: address = @0x1`, and then re-use them across tests.